plugin/trino-bigquery/pom.xml

```diff
@@ -426,8 +443,10 @@
                         <groupId>org.apache.maven.plugins</groupId>
                         <artifactId>maven-surefire-plugin</artifactId>
                         <configuration>
+                            <argLine>--add-opens=java.base/java.nio=ALL-UNNAMED</argLine>
```

Comment:
You can move this to a new profile called `arrow` and add it only there - that way we can be sure that only the Arrow part of code needs the add-opens.

Reply:
Can be follow-up.